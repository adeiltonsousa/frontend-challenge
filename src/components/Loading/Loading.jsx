import React from 'react'

import styles from './Loading.module.css'

const Loading = () => <p className={styles.loading}>Carregando...</p>

export default Loading
