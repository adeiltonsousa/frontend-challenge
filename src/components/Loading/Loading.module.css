.loading {
  left: 50%;
  top: 50%;
  position: absolute;
  font-size: 1rem;
  color: var(--color-purple);
}
