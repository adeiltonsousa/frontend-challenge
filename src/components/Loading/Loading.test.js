import React from 'react'
import { render } from '@testing-library/react'

import Loading from './Loading'

describe('<Loading />', () => {
  it('should render Loading component', () => {
    const { container, findByText } = render(<Loading />)

    expect(findByText('Carregando...'))
    expect(container).toMatchSnapshot()
  })
})
