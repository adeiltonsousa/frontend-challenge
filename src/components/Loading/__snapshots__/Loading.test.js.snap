// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render Loading component 1`] = `
<div>
  <p>
    Carregando...
  </p>
</div>
`;
